c++/cavl.hpp

```diff
@@ -175,63 +172,37 @@ class Node
     static auto traverse(Derived* const root, const Vis& visitor, const bool reverse = false)  //
         -> std::enable_if_t<!std::is_void<R>::value, R>
     {
-        if (Node* const n = root)
```

Comment:
What do you think about renaming public `Tree::travese` & protected `Node::traverse` → `Xxx::inOrderTraverse` ?